docs/source/json-api/production-setup.rst

```diff
@@ -204,13 +204,24 @@ Users may consider running PostgreSQL backend in a `high availability configurat
 The benefits of this are use-case dependent as this may be more expensive for
 smaller active contract datasets, where re-initializing the cache is cheap and fast.
 
-Finally we recommend using app orchestration utilities which monitor the health of the service
-and perform subsequent operations to ensure availability. These utilities can use the
+Finally we recommend using orchestration systems or load balancers which monitor the health of
+the service and perform subsequent operations to ensure availability. These systems can use the
 `healthcheck endpoints <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`__
 provided by the *HTTP JSON API* server. This can also be tied into supporting arbitrary
 autoscaling implementation to ensure minimum number of *HTTP JSON API* servers on
 failures.
 
+Setup the HTTP JSON API Service to work with Highly Available Participants
+--------------------------------------------------------------------------
+
+In case the participant node itself is configured to be highly available, depending on the setup you might want
+to choose different strategies. In most setups, including those based on Canton, you'll likely have an active
+participant node whose role can be taken over by a passive node in case the currently active one drops. Just as
+for the *HTTP JSON API* itself, you can again use orchestration systems or load balancers to monitor the status
```

Comment:
can omit the word "again" here.

Reply:
Addressed by https://github.com/digital-asset/daml/commit/291cd7bce25775f9ae1230306503b7308b762a39